.github/workflows/report-deprecation.yml

```diff
@@ -0,0 +1,30 @@
+name: Report Deprecation
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '20.12.2'
```

Comment:
Do we need a specific node version? Or is 20.x enough?

Reply:
I just used what I had on my system, I think 20.x is indeed enough